src/components/organisms/PublicHeader.vue

```diff
@@ -0,0 +1,34 @@
+<script setup>
+import useLayout from '~/composables/layout'
+import { APP_TITLE } from '~/constants'
+import DcBtn from '../atoms/DcBtn.vue';
+
+const { toggleCartDrawer, toggleMenuDrawer } = useLayout()
+</script>
+
+<template>
+  <QHeader elevated class="bg-dark text-white" style="height: 80px" height-hint="80">
+    <QToolbar class="full-height">
+      <QToolbarTitle>
+        <QAvatar>
+          <img src="/favicon.svg">
+        </QAvatar>
+        {{ APP_TITLE }}
```

Comment:
I think it might not be a good practice to make a direct reference to a env variable. What about setting a computed or constant property to assign this value?

Reply:
That `APP_TITLE` is imported from the `src/constants` folder I already defined it there as it won't change in the application. Should I change this folder, or the way it is written, maybe?